src/routes/popup/settings/wallets/[address]/qr.tsx

```diff
@@ -108,7 +108,9 @@ export function GenerateQRView({ params: { address } }: GenerateQRViewProps) {
               <ContentWrapper>
                 <Section>
                   <QRCodeWrapper size={275}>
-                    <QRCodeLoop frames={frames} fps={5} size={275} />
+                    <div style={{ backgroundColor: "#fff" }}>
```

Comment:
```suggestion
                    <div style={{ backgroundColor: "#fff", padding: "12px", borderRadius: "12px" }}>
```
That way we have some white padding and border radius around the QR Code